Sorting/insertionSort.js

```diff
@@ -0,0 +1,52 @@
+//insertion sort works by defining the first element in the array as sorted.
+//then inserting each subsequent element in the array in the correct location, progressively building a sorted array.
+
+
+function insertionSort(arr) {
+  var sortedPortion = arr.slice(0,1)
```

Comment:
rather than creating a new array, it should be possible to do the sort _in place_: reuse the provided array without allocating a new one.